src/join-the-team.ejs

```diff
@@ -20,9 +20,12 @@
                     problems and objectives that they are given. All
                     development is based in London in the same offices as
                     our journalists and editors.</p>
+
+            <p>The Guardian aims to ensure that all job applicants are treated fairly regardless of race (which includes nationality and ethnic or national origin), disability, sex, gender re-assignment, marital or civil partnership status, sexual orientation, perceived or actual age, maternity, pregnancy and religion or belief (hereafter referred to as protected characteristics).</p>
```

Comment:
We don't actually refer to `protected characteristics` anywhere, so I'd drop the parenthesised bit.